client-spark/common/src/test/java/org/apache/uniffle/shuffle/manager/ShuffleManagerServerFactoryTest.java

```diff
@@ -17,26 +17,28 @@
 
 package org.apache.uniffle.shuffle.manager;
 
-import org.junit.jupiter.api.Test;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 import org.apache.uniffle.common.config.RssBaseConf;
 import org.apache.uniffle.common.rpc.ServerType;
 
-import static org.junit.jupiter.api.Assertions.assertThrows;
-
 public class ShuffleManagerServerFactoryTest {
-  @Test
-  public void testShuffleManagerServerType() {
+  private static Stream<Arguments> shuffleManagerServerTypeProvider() {
+    return Stream.of(Arguments.of(ServerType.GRPC_NETTY), Arguments.of(ServerType.GRPC));
```

Comment:
I liked the `ServerType.values()` that you had earlier, because this automatically tests all types if more are added:
```suggestion
    return Arrays.stream(ServerType.values());
```

Reply:
Done.